helm_deploy/whereabouts-api/templates/_envs.tpl

```diff
@@ -8,7 +8,7 @@ env:
     value: "{{ .Values.image.port }}"
 
   - name: SPRING_PROFILES_ACTIVE
-    value: "postgres"
+    value: "postgres,logstash"
```

Comment:
logstash profile outputs as json so suitable for app insights.  This is also the one that triggers the `aiAppender`